modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/volumetricdatalayer/v0.1.0.json

```diff
@@ -0,0 +1,148 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/atlas/volumetricdatalayer/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/transformableobject/v1.0.0",
```

Comment:
{{base}}/schemas/neurosciencegraph/commons/transformableobject/v1.0.0 should be part of this PR or is it in another PR ?